bin-src/init.ts

```diff
@@ -0,0 +1,142 @@
+import boxen from "boxen";
+import { execaCommand } from 'execa';
+import { findUp } from 'find-up';
+import { writeFile } from 'jsonfile';
+import meow from 'meow';
+import prompts from 'prompts';
+import type { PackageJson } from 'read-pkg-up'
+import readPkgUp from 'read-pkg-up';
+import { getCliCommand, parseNi } from '@antfu/ni';
+
+import noPackageJson from '../node-src/ui/messages/errors/noPackageJson';
+
+export const TestFramework = {
+    STORYBOOK: 'storybook',
+    PLAYWRIGHT: 'playwright',
+    CYPRESS: 'cypress'
+};
+type TestFrameworkType = typeof TestFramework[keyof typeof TestFramework];
+
+export const getPackageManagerInstallCommand = async (args: string[]) => {
+    return await getCliCommand(parseNi, args, { programmatic: true });
+};
+
+const getSBConfigFilePath = async () => {
+    // Walks up directory tree to find nearest Storybook config file.
+    return await findUp(['.storybook/main.ts', '.storybook/main.js', '.storybook/main.tsx', '.storybook/main.jsx', '.storybook/main.mjs', '.storybook/main.cjs'])
+}
+
+export const addChromaticScriptToPackageJson = async ({ packageJson, packagePath }) => {
+    try {
+        const json = {
+            ...packageJson,
+            scripts: {
+                ...packageJson?.scripts,
+                chromatic: `npx chromatic@latest`
+            }
+        }
+        await writeFile(packagePath, json, { spaces: 2 });
+    } catch (e) {
+        console.warn(e)
+    }
+}
+
+export const createChromaticConfigFile = async ({configFile, buildScriptName = null}) => {
+    await writeFile(configFile, {
+        ...(buildScriptName && {
+            buildScriptName
+        }),
+        skip: "dependabot/**"
+    });
+}
+
+export const installArchiveDependencies = async (packageJson: PackageJson, testFramework: TestFrameworkType) => {
+    let installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, 'storybook@next', '@storybook/addon-essentials@next', '@storybook/server-webpack5@next']
+    const sbConfigPath = await getSBConfigFilePath()
+    const sbVersion = packageJson?.devDependencies?.storybook || packageJson?.dependencies?.storybook
+    if(sbConfigPath && sbVersion) {
+        installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, `@storybook/server-webpack5@${sbVersion}`]
+    }
+    const installCommand = await getPackageManagerInstallCommand(installArgs)
+    await execaCommand(installCommand)
+}
+
+const intializeChromatic = async ({testFramework, packageJson, packagePath}: {testFramework: TestFrameworkType, packageJson: PackageJson, packagePath: string }) => {
+    await addChromaticScriptToPackageJson({ packageJson, packagePath });
+    await createChromaticConfigFile({configFile: 'chromatic.config.json'})
```

Comment:
This `chromatic.config.json` file isn't currently used/needed for E2E. Could we skip generating this file in those cases for now?

Reply:
The Config file or CLI flags are needed for configuration of Chromatic CLI/Action.

We are trying to promote the config file moving forward as the way to configure Chromatic. In the future, this file would automatically be configured with setting that we recommend like the one here.

In this case, if we take this out, Chromatic would run on any dependabot PRs that open. Customers complain about this often enough that we codify this in our docs.

Reply:
TLDR: If you're running Chromatic CLI, you will need either CLI flags or the config.